android/src/main/java/io/mosip/tuvali/openid4vpble/events/withArgs/VerificationStatusEvent.kt

```diff
@@ -0,0 +1,16 @@
+package io.mosip.tuvali.openid4vpble.events.withArgs
+
+import io.mosip.tuvali.openid4vpble.events.EventWithArgs
+
+class VerificationStatusEvent(private val status: VerificationStatus): EventWithArgs {
+  override fun getData(): HashMap<String, String> {
+    return hashMapOf("status" to status.value)
+  }
+
+  override val type = "onVerificationStatusReceived"
+
+  enum class VerificationStatus(val value: String) {
+    ACCEPTED("0"),
```

Comment:
Use integer Enum values. React native layer can have glue code to do conversion from "ACCEPTED" to 0 and "REJECTED" to 1

Reply:
We will converted this type to integer in other pr. as this pr has changes which doesn't support integer